src/main/java/edu/harvard/iq/dataverse/engine/command/impl/FinalizeDatasetPublicationCommand.java

```diff
@@ -220,7 +235,7 @@ private void exportMetadata(Dataset dataset, SettingsServiceBean settingsService
             ExportService instance = ExportService.getInstance(settingsServiceBean);
```

Comment:
I don't see why not; maybe handle it in the same issue that you have opened for moving the indexing into onSuccess?
Otherwise, I made that exception catch more general (that appeared to be the reason a recent problem with OpenAire export was resulting in a fatal error when publishing... and the code below suggested we never wanted to treat a failure to export as fatal).

Reply:
I feel like export actually already is in onSuccess anyway? so are we exporting twice?? I think it's fine to handle in the other issue.

Reply:
Uh, you're right - it appears to already be in onSuccess.
So, what were you proposing then?
(are you still waiting for something from me for this PR?)

Reply:
If we're sure the onsuccess export does everything the same, I would remove from here. If we're not, we can leave and add as a comment to the other issue to review and clean up then.

(other than that, can move to QA)

Reply:
But, this is the method that's called from onSuccess. And it doesn't seem to be called from anywhere else... It's entirely possible that I'm missing something obvious.

Reply:
Oh. Not you, me --- the way it was truncated in github, it looked to me as if it were in execute. My mistake. (After expanding, yes, it is in onSuccess - which is good, since it's what we expected. Sorry for the noise.